tests/data/mattermost/mattermost_posts.json

```diff
@@ -96,7 +96,17 @@
             "type": "",
             "props": {},
             "hashtags": "",
-            "pending_post_id": ""
+            "pending_post_id": "",
+            "metadata": {
```

Comment:
Can we add another item, so we check that the loop creating the images data works as expected?